larq/models.py

```diff
@@ -110,6 +110,14 @@ def memory(self) -> int:
     def fp_equivalent_memory(self) -> int:
         return 32 * self.count
 
+    @property
+    def int8_fp_weights_memory(self) -> int:
+        """Count any 32- or 16-bit weights as 8 bits instead."""
+
+        if self.bitwidth == 32 or self.bitwidth == 16:
```

Comment:
Would consider 
```suggestion
        if self.bitwidth > 8:
```
It doesn't really make sense to assume 32 bit weights will be quantized to 8 bits, while 24 bit weights remain 24.